src/Tools/dotnet-monitor/Extensibility/ExtensionException.cs

```diff
@@ -45,6 +45,11 @@ public static void ThrowInvalidManifest(JsonException ex)
             throw new ExtensionException(ex.Message, ex);
         }
 
+        public static void ThrowFileNotFound(string extensionName, string path)
```

Comment:
Also, small nit for readability: consider reordering the parameters so that they match the `ErrorMessage_ExtensionFileNotFound`'s order - I originally thought they were backward when we call this method, but then I realized this method is flipping their order.

Reply:
Since we don't have nullability checks enabled, the annotations don't do anything. I don't think this will be enabled any time soon unless someone makes a concerted effort to fixed up the entire tool.

Reply:
I'm not concerned about the parameter ordering in comparison to the format string. The reason being that if we want to change what the format string states, I don't think we should reorder the parameters again to match it; that would be way too much refactoring.